packages/usa-breadcrumb/src/styles/_usa-breadcrumb.scss

```diff
@@ -185,6 +185,14 @@ $breadcrumb-back-icon-aspect: (
   }
 }
 
+.usa-breadcrumb__link[aria-current="page"] {
+  @include add-link-styles(
+    $theme-text-color,
+    $hover: $theme-text-color,
+    $active: $theme-text-color
+  );
+}
+
```

Comment:
**Question:** Wouldn't this be considered an a11y failure since there is no visual indication that it's a link?

From my understanding, [this is an example of a failure to comply with criterion 1.4.1](https://www.w3.org/TR/WCAG20-TECHS/F73.html). WCAG recommends [two visual indicators](https://www.w3.org/TR/WCAG20-TECHS/G182.html) that it is different from normal body text.

CC: A11y team @alex-hull @amyleadem

Reply:
It is focusable, however, and is read aloud in screen readers as "current".

Reply:
@amyleadem the thing is with this example is that I think the bolded text counts as a "visual indicator other than color"

It's similar to [this example](https://arc.net/l/quote/cxhitkjm) provided by W3C

> A news site lists links to the articles appearing on its site. Additional information such as the section the article appears in, the time the article was posted, a related location or an indication that it is accompanied by live video appears in some cases. The links to the articles are in a different color than the additional information but **the links are not underlined, and each link is presented in a larger font** than the rest of the information so that users who have problems distinguishing between colors can identify the links more easily.

Reply:
I'm uncertain if this is _only_ required if a different color is used, and I'm having trouble finding a criterion that denotes a link must have visual indicators. I feel like we've identified this as an accessibility issue in the past 🤔